ios/FirebaseContinue/Example/FirebaseContinue/main.m

```diff
@@ -15,8 +15,20 @@
 //
 
 @import UIKit;
+
 #import "FCNAppDelegate.h"
 
+/**
+ * Important Note: This is not a working example of using Firebase Continue.
+ *
+ * The Firebase Continue for iOS library is planned to eventually be available via Cocoapods
+ * (rather than being required to manually copy-paste its source into your project), so this
+ * Cocoapods-provided development environment lays the groundwork for that.
+ *
+ * For an actual, working sample of Firebase Continue, see the samples/ios subdirectory
+ * from the root of the repository at:
+ * https://github.com/firebase/firebase-continue/tree/master/samples/ios
+ */
 int main(int argc, char * argv[])
```

Comment:
Since this isn't addressed in the ObjC style guide we should fallback to the C++ guide, which says that the * can either go with the type or the variable name but spaces on both sides are not allowed: https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

To match the rest of your code I recommend char *argv[].